README.md

```diff
@@ -0,0 +1,87 @@
+# Kibana 6
+
+This role sets up a running kibana instance and configures it.
+
+
+## Requirements
+
+None
+
+
+## Role Variables
+
+For variables please see the [kibana documentation](https://www.elastic.co/guide/en/kibana/current/settings.html)
+
+### Mandatory variables
+Some variables are only possible to configure together they are also listed below
+
+| Name                                    |         Required         | Default | Description                                                                                                                                                            |    |
+|:----------------------------------------|:------------------------:|:--------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|:---|
+| `kibana_elasticsearch_url `             |    :heavy_check_mark:    |         | The URL of the Elasticsearch instance to use for all your queries                                                                                                      |    |
```

Comment:
Are these really my queries? What if somebody else logs on and performs queries?

Reply:
Same comment as in the Kibana docs